### ISSUE TITLE
Add default setting for min/max values accuracy

### ISSUE THREAD

**Issue by lnicola** [qgis/QGIS#33261]
Basically same as https://github.com/qgis/QGIS/issues/20181. It's possible to toggle this from Estimate to Actual on each raster, but the default is still Estimate, and there seems to be no way to configure it from settings.

**Comment by elvilanova** [comment]
Agree. It's the same as #20181. If one is performing several raster analysis has to go to each raster and change its accuracy from Estimate to Actual. The default Estimate is not desirable, it can lead to misinterpretations. I guess the fastest solution is to change QGIS default behaviour, that is, to be Actual. In the long term, it could be solved similarly to how it is possible for 'Limits (minimum/maximum)' in 'Settings > Options > Rendering', there could be a list allowing a user to choose raster accuracy default behaviour.

**Comment by elvilanova** [comment]
![Raster_Stats](https://github.com/user-attachments/assets/9d93187f-b1b3-4fb2-b4c1-a319c2823508)
This mocked image is somehow what is intended with this issue, as well as with #20181. Default behavior should be 'Actual' and give the user the ability to choose Accuracy mode default behavior in the 'Settings > Options > Rendering > Raster' options menu.
